packages/teams_memory/teams_memory/utils/teams_bot_middlware.py

```diff
@@ -145,11 +150,17 @@ async def _augment_context(self, context: TurnContext):
             ScopedMemoryModule(
                 self.memory_module,
                 users_in_conversation_scope,
-                conversation_ref_dict.conversation.id,
+                (
+                    conversation_ref_dict.conversation.id
+                    if conversation_ref_dict and conversation_ref_dict.conversation
+                    else ""
```

Comment:
Hm, don't default to "". Log an error. This should be an unexpected case.

Reply:
(right?)

Reply:
What do you want to set the return value to? Has to be a string. Happy to add the error.